locallibrary/catalog/models.py

```diff
@@ -1,3 +1,74 @@
 from django.db import models
 
+
 # Create your models here.
+class Genre(models.Model):
+    """Model representing a book genre."""
+    name = models.CharField(max_length=200, help_text='Enter a book genre (e.g.Science Fiction)')
```

Comment:
Django I18n để translate text phục vụ tính năng đa ngôn ngữ ứng dụng e nhé
Translate tất cả nội dung text của option `help_text`
https://docs.djangoproject.com/en/4.2/topics/i18n/translation/#internationalization-in-python-code

Sửa tương tự trong gile